src/uct/ib/rdmacm/rdmacm_cm.h

```diff
@@ -17,8 +17,17 @@
 typedef struct uct_rdmacm_cm {
     uct_cm_t                  super;
     struct rdma_event_channel *ev_ch;
+
+    struct {
+        struct sockaddr       *src_addr;
```

Comment:
why does it need to be pointer?
can use AF_UNSPEC to mark it as invalid to pass NULL to rdmacm

Reply:
IMHO better to initialize once and reuse the exact value on control path every time, also NULL-pointer looks much simpler

Reply:
IMO it could be just "struct sockadd_storage" here which is initialized once, would be simpler to avoid malloc/free/free in err flow